auction-deploy/auction_deploy/core.py

```diff
@@ -0,0 +1,132 @@
+import json
+import pkg_resources
+from typing import Dict, NamedTuple
+from collections import namedtuple
+
+from eth_keyfile import extract_key_from_keyfile
+from deploy_tools.deploy import send_function_call_transaction, deploy_compiled_contract
+
+
+class AuctionOptions(NamedTuple):
+    start_price: int
+    auction_duration: int
+    number_of_participants: int
+    release_block_number: int
+
+
+DeployedAuctionContracts = namedtuple("DeployedContracts", "locker slasher auction")
+
+
+def load_contracts_json() -> Dict:
+    resource_package = __name__
+    stream = pkg_resources.resource_stream(resource_package, "contracts.json")
+    json_string = stream.read().decode()
+    stream.close()
+    return json.loads(json_string)
+
+
+def decrypt_private_key(keystore: str, password: str) -> bytes:
+    return extract_key_from_keyfile(keystore, password.encode("utf-8"))
+
+
+def increase_transaction_options_nonce(transaction_options: Dict) -> None:
+    if "nonce" in transaction_options:
+        transaction_options["nonce"] = transaction_options["nonce"] + 1
+
```

Comment:
It might be better to throw an exception if "nonce" isn't contained in `transactions_options`. I'm not sure however how this function is being used,so it might also be done on purpose. In that case,please add a docstring.

Reply:
Added a doc string